### DISCUSSION TITLE
Adding a new UnistylesRuntime.statusBar.setBackgroundColor method

### DISCUSSION THREAD

**Discussion by VladyslavMartynov10** [jpudysz/react-native-unistyles#192] [Feature Requests]
Hi @jpudysz,

Thank you for creating such a cool and unified styling library that enables significant enhancements with robust performance. Is it possible to add a new method to the UnistylesRuntime host object?

`UnistylesRuntime.statusBar.setBackgroundColor`

I believe this can only be handled on Android. Thank you in advance!

**Comment by jpudysz** [comment]
Hey 👋
Thank you for your feedback, really appreciate it!

It's already implemented 😅
Check out the docs:

https://reactnativeunistyles.vercel.app/reference/theming/#update-statusbar-color-during-runtime

**Reply by VladyslavMartynov10** [reply]
Thanks!
